crew

```diff
@@ -728,45 +732,45 @@ end
 
 def remove (pkgName)
 
-  #make sure the package is actually installed
+  # Make sure the package is actually installed
   unless @device[:installed_packages].any? { |pkg| pkg[:name] == pkgName }
     puts "Package #{pkgName} isn't installed.".lightred
     return
   end
 
-  #if the filelist exists, remove the files and directories installed by the package
+  # If the filelist exists, remove the files and directories installed by the package
   if File.file?("#{CREW_CONFIG_PATH}meta/#{pkgName}.filelist")
     Dir.chdir CREW_CONFIG_PATH do
 
-      #remove all files installed by the package
+      # Remove all files installed by the package
       File.open("meta/#{pkgName}.filelist").each_line do |line|
         begin
           File.unlink line.chomp
-        rescue => exception #swallow exception
+        rescue => exception # swallow exception
```

Comment:
__[RuboCop]__

Lint/UselessAssignment: Useless assignment to variable - `exception`.


<!-- Comment by [SideCI](https://sideci.com) -->